monad-rpc/src/handlers/debug.rs

```diff
@@ -567,6 +624,22 @@ pub async fn decode_call_frame<T: Triedb>(
         .flatten()
         .collect::<Vec<_>>();
 
+    for frame in call_frames.iter_mut() {
+        if tracer.config.with_log {
+            // If logs were requested and there were none stored with the call
```

Comment:
Is there a reason we return an error? Anvil does not return an error if the transaction's call traces do not have logs